pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2029,7 +2029,13 @@
       }
     },
     "System.IO.Ports": {
-      "InboxOn": {},
+      "StableVersions": [
```

Comment:
cc: @ericstj so that he can take a quick look at this. Section looks good to me but I thought we only needed the InboxOn part, so better if he can take a look.